### PR TITLE
Brainstorming with a map

### PR DESCRIPTION
**Pull request by 2e2a**
Depends on: https://github.com/liqd/adhocracy4/pull/57

Notes:
- The CSS change seems to break the categories widget.

### CODE REVIEW

**Comment by xi** [comment]
5eb3b3a is incomplete. The data still needs to be JSON encoded. It may be good to define a template filter for that which could also be used in core.

**Comment by xi** [comment]
> The CSS change seems to break the categories widget.

@vellip this is about removing the global `max-width 100%`. Can you please look into that? I also noticed that the image upload is broken.

**Comment by vellip** [comment]
the max-width regressions should be fixed now.

File: apps/mapideas/templates/meinberlin_mapideas/mapidea_detail.html [hunk 1]
```diff
@@ -69,8 +69,8 @@ <h1 class="resource-header__title">{{ object.name }}</h1>
                     style="height: 300px;"
                     data-map="display_point"
                     data-baseurl="{{ map_url }}"
-                    data-point="{{ object.point }}"
-                    data-polygon="{{ object.module.settings_instance.polygon }}"
+                    data-point="{{ point }}"
+                    data-polygon="{{ polygon }}"
```
Comment (xi) [inline apps/mapideas/templates/meinberlin_mapideas/mapidea_detail.html]: These values still need JSON encoding to work reliably.

Comment (vellip) [inline apps/mapideas/templates/meinberlin_mapideas/mapidea_detail.html]: That's happening in core. https://github.com/liqd/adhocracy4/pull/57/commits/05cb62a7cd40cf4a540b1319a41ba1b16473220b

**Pull request merged**